### DISCUSSION TITLE
Endless synchronization

### DISCUSSION THREAD

**Discussion by TraceUser** [Chia-Network/chia-blockchain#14160] [Support]
I have to use Chia Wallet in Wallet mode. I used the wallet once and made one transaction. Now I go to my wallet and my wallet has infinite sync 
![изображение](https://user-images.githubusercontent.com/120669481/207917946-77792b2c-e5aa-4cc1-b47f-db67f8e8ec7f.png)

**Comment by MumfMeisterT** [comment]
Which Chia version are you running this on? 
Can you send us logs for this, and then try deleting your wallet folder and restarting?
